core/src/main/resources/chart/article/index.scala.html

```diff
@@ -1,6 +1,8 @@
+@import com.gu.contentatom.renderer.utils.Implicits._
 @(atom: com.gu.contentatom.thrift.Atom
 , data: com.gu.contentatom.thrift.atom.chart.ChartAtom
 )(implicit conf: com.gu.contentatom.renderer.ArticleConfiguration)
 
-
-<iframe srcdoc="@atom.defaultHtml" frameborder="0"></iframe>
+<div class="atom atom--chart">
+  <iframe name="@atom.id" class="atom__iframe" srcdoc="@data.newDefaultHtml(atom.defaultHtml)" src="#" frameborder="0"></iframe>
```

Comment:
What's the role of the `src` attribute here?

Reply:
Without it I had `SecurityException`s threwn when trying to access `window.name` 😭

Reply:
oh, that's really odd 🤔